Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+#!groovy
+
+def buildSite() {
+    stage ('build') {
+      try {
+        sh 'bin/build.sh'
+      } catch(err) {
+          sh "bin/irc-notify.sh --stage 'build " + env.BRANCH_NAME + "' --status 'failed'"
+        throw err
+      }
+    }
+}
+
+def syncS3(String bucket) {
+    stage ('s3 sync') {
+        try {
+          sh "cd release && aws s3 sync . s3://" + bucket +" --acl public-read --delete --profile irl --exclude 'docs/*'"
```

Comment:
Note the `--exclude docs`: the dir that you've built locally won't affect stage or prod, only the github pages which are optional now.